src/msr_data_knl.cpp

```diff
@@ -381,7 +386,8 @@ namespace geopm
                     "units":     "watts",
                     "scalar":    1.25e-1,
                     "writeable": false,
-                    "aggregation": "sum"
+                    "aggregation": "sum",
+                    "description": "Maximum power to stay within the thermal limits based on the design (TDP)."
```

Comment:
Maximum power to stay within the thermal limits in the design specification (TDP).